ProcessLib/TES/TESLocalAssembler-impl.h

```diff
@@ -0,0 +1,255 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#ifndef PROCESS_LIB_TES_FEM_IMPL_H_
+#define PROCESS_LIB_TES_FEM_IMPL_H_
+
+
+#include "MaterialsLib/Adsorption/Adsorption.h"
+#include "NumLib/Fem/FiniteElement/TemplateIsoparametric.h"
+#include "NumLib/Fem/ShapeMatrixPolicy.h"
+#include "NumLib/Function/Interpolation.h"
+#include "ProcessLib/Utils/InitShapeMatrices.h"
+
+#include "TESLocalAssembler.h"
+#include "TESReactionAdaptor.h"
+
+namespace
+{
+enum class MatOutType { OGS5, PYTHON };
+
+const MatOutType MATRIX_OUTPUT_FORMAT = MatOutType::PYTHON;
+
+template<typename Mat>
+void
+ogs5OutMat(const Mat& mat)
```

Comment:
Actually it's used below to do debug output. I can move it to some core debug function file later on. Gonna add a TODO comment.